CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -100,6 +98,13 @@ void Kartaclysm::StateRacing::BeginRace()
 	HeatStroke::GameObject* pTrack = m_pGameObjectManager->CreateGameObject(m_mContextParams.at("TrackDefinitionFile"), "Track");
 	ComponentTrack* pTrackComponent = static_cast<ComponentTrack*>(pTrack->GetComponent("GOC_Track"));
 
+	// TODO: weird issue with Racer transforms unless m_vWorldTranslation is changed on walls
```

Comment:
Cool.  Maybe update the TODO to be more descriptive about the problem, but otherwise this is good